menpofit/result.py

```diff
@@ -595,6 +595,8 @@ def __str__(self):
         if self.gt_shape is not None:
             if self.initial_shape is not None:
                 out += "\nInitial error: {:.4f}".format(self.initial_error())
+                if isinstance(self, ParametricIterativeResult) or isinstance(self, MultiScaleParametricIterativeResult):
```

Comment:
I see other people warn about using hasattr() unless the code in Python3-only.

But I'm just a week in Python so it's hard for me to make my opinion on the issue therefor the commit follows.